browser-tests/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why has this file been added again?  A file with the same name was added previously which includes rules that must be followed before committing code. What is the use of adding this additionally?

Reply:
Hi @rama1990br , the reason this is added here is because we're writing our JS for the application using older JS version (ES-5), but our tests are using the latest versions of JavaScript (ES6 & ES7). The reason for doing this is because writing tests is extremely simple when you can use the latest language features. In time, we will also migrate the application code to ES-6 and ES-7.

The original `.eslintrc` file contains the rules appropriate for ES-5. This file is added under the `browse-tests` directory because the settings in this file need to be given precedence when writing tests. `eslint` always respects the "closest" `.eslintrc` file. So, we need to have this in order to prevent git pre-commit hook errors.